garage/np/algos/base.py

```diff
@@ -1,7 +1,10 @@
+from garage.core import Serializable
+
+
 class Algorithm:
     pass
 
 
-class RLAlgorithm(Algorithm):
+class RLAlgorithm(Algorithm, Serializable):
```

Comment:
This makes me uncomfortable. Inheriting from Serializable has a lot of effects, and many of them are surprising. I would much rather each algorithm that needs this inherit it itself.

I also don't think most algorithms should need to inherit Serializable, since they should already be picklable.

Reply:
I removed Serializable from all algorithms.